sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -327,13 +348,13 @@ def __init__(
         self._encryption_options = encryption_options or {}
         self._progress_hook = kwargs.pop('progress_hook', None)
         self._request_options = kwargs
-        self._location_mode = None
+        self._location_mode: Optional[str] = None
         self._download_complete = False
-        self._current_content = None
-        self._file_size = None
-        self._non_empty_ranges = None
+        self._current_content: bytes = b''
```

Comment:
Can you check to make sure that `b''` evaluates to False? If so, this is fine.

Reply:
Yep, evalauates to `False`